pkg/apis/intevents/v1alpha1/brokercell_defaults.go

```diff
@@ -23,15 +23,15 @@ import (
 )
 
 const (
-	avgCPUUtilizationFanout  int32 = 95
+	avgCPUUtilizationFanout  int32 = 30
```

Comment:
I am concerned if we are overfitting to solve rapid request spikes. If I have a gradual request increase that just causes fanout to use CPU at, say, 40%, this will result in 2 pods with low CPU utilization. Is my understanding correct?

Reply:
Good point, underutilization of resources in such cases was my major concern as well. There are some thoughts on this in the "Fanout with Autoscaling" doc. Overall, with limited control over scale out velocity, we run into this trade-off between performance at load and utilization of resources under lower load.  In 1.18+, this can be better addressed with recently introduced [scale out behavior](https://github.com/kubernetes/enhancements/blob/master/keps/sig-autoscaling/20190307-configurable-scale-velocity-for-hpa.md), but for now, which of these would you rather prefer:
1. 50% threshold with ~6-7 minutes cold start, but lower resource utilization at lower load;
1. 30% threshold with ~2-3 minutes cold start (~50% better), but considerably higher resource utilization at lower load;

    - (For a light load, like under 10Mb/s delivery rate, the scale out is not triggered, so, both options are on par;

I guess it's all application/customer dependent, but I would love to hear more thoughts on our expected default/common case.

Reply:
We don't know for sure until we have more customer data. I personally prefer a higher number as I think use cases that require aggressive scaling is less common. 

@grantr @yolocs thoughts on this?

Reply:
When doing load test, we tend to think more about worst case scenarios, which could be a lot of worse than the real cases. I agree a less aggressive scaling could be a good starting point.

Reply:
Makes sense. Ok, I think we can proceed as following:
1) For the default configuration, I'll proceed with option #1 with higher threshold;
2) Those seeking better performance at high RPS would have two workarounds: lowering this threshold or increasing minReplicas. We may provide guidance on configurability and these trade-offs in the documentation;

Sounds good?

Reply:
I hope we can achieve a higher CPU utilization like 70. But I can be convinced with your proposal given the constraints we have today.

Reply:
Now that we know the effect of changing this number, it will be relatively simple to tune in response to user feedback.

Reply:
We've experimented with higher thresholds -- other factors start weighting in, e.g. there are increasing risks that autoscaling occasionally won't be triggered at all, which would result in the throughput considerably below the target. If we could include an additional robust metric like the number of messages in PubSub backlog, I guess, that would have addressed that risk.